packages/components/src/components/hds/side-nav/index.ts

```diff
@@ -72,6 +77,23 @@ export default class HdsSideNav extends Component<HdsSideNavSignature> {
     registerDestructor(this, (): void => {
       this.removeEventListeners();
     });
+
+    if (args.ariaLabel !== undefined) {
+      deprecate(
+        'The `@ariaLabel` argument for "Hds::SideNav" has been deprecated. It is replaced by aria-labelledby and aria-expanded on the toggle button',
+        false,
+        {
+          id: 'hds.sidenav',
+          until: '5.0.0',
+          url: 'https://helios.hashicorp.design/components/sidenav?tab=version%20history#4130',
+          for: '@hashicorp/design-system-components',
+          since: {
+            available: '4.13.0',
```

Comment:
suggestion: next release is going to be 4.14 (needs updated in the website docs as well)
```suggestion
            available: '4.14.0',
```